docs/dev/reference/changelog.md

```diff
@@ -42,6 +42,13 @@ date: "2024-09-18"
 # updated: ""  # When the content was last entirely checked
 ---
 
+{{% changelog color="removed" title="Frame tab removed, use Add Frame button" date="2025-05-15" %}}
+
+The frame tab no longer exists in the Viam app.
+To add a frame to a component, click the **+ Add Frame** button on the component's configuration card.
```

Comment:
```suggestion
To add a frame to a component, click the **+ Add Frame** button on a component's configuration card.
```

Reply:
We already introduced "a component" in the first part of the sentence so it'd be odd to say "a" again I think?